src/systemtest/java/reposense/ConfigSystemTest.java

```diff
@@ -58,25 +58,26 @@ public void tearDown() throws Exception {
      */
     @Test
     public void testSinceBeginningDateRange() throws Exception {
-        generateReport(getInputWithDates(SinceDateArgumentType.FIRST_COMMIT_DATE_SHORTHAND, "2/3/2019"));
+        generateReport(getInputWithDates(SinceDateArgumentType.FIRST_COMMIT_DATE_SHORTHAND, "2/3/2019"), false);
         Path actualFiles = loadResource(getClass(), "sinceBeginningDateRange/expected");
         verifyAllJson(actualFiles, FT_TEMP_DIR);
     }
 
     @Test
     public void test30DaysFromUntilDate() throws Exception {
-        generateReport(getInputWithUntilDate("1/11/2017"));
+        generateReport(getInputWithUntilDate("1/11/2017"), false);
         Path actualFiles = loadResource(getClass(), "30daysFromUntilDate/expected");
         verifyAllJson(actualFiles, FT_TEMP_DIR);
     }
 
     /**
-     * System test with a specified since date and until date.
+     * System test with a specified since date and until date, with the last modified date time in each
+     * line of code.
      */
     @Test
-    public void testDateRange() throws Exception {
-        generateReport(getInputWithDates("1/9/2017", "30/10/2017"));
-        Path actualFiles = loadResource(getClass(), "dateRange/expected");
+    public void testDateRangeWithModifiedDateTimeInLines() throws Exception {
```

Comment:
Created this system test that merges with the previous `testDateRange` as it feels redundant to have another system test that may take up our build time and space.